Assets/Scripts/Pathfinding.cs

```diff
@@ -1,18 +1,32 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using Unity.AI;
+using UnityEngine.AI;
 
 public class Pathfinding : MonoBehaviour
 {
+    [SerializeField]
+    private GameObject Player;
+    private Vector3 Target;
```

Comment:
Target should probably be a public Transform so that we can test it in the editor play mode.